BC.HelperFunctions.ps1

```diff
@@ -118,6 +118,7 @@ function Get-ContainerHelperConfig {
             "IsGitHubActions" = ($env:GITHUB_ACTIONS -eq "true")
             "IsAzureDevOps" = ($env:TF_BUILD -eq "true")
             "IsGitLab" = ($env:GITLAB_CI -eq "true")
+            "useApproximateVersion" = $true
```

Comment:
Should be an opt-in and not an opt-out IMO. At least to begin with. If we find a bug in it the next few weeks it won't impact everyone